### PR TITLE
Remove php snaptest from producers.md

### PR DESCRIPTION
**Pull request by imme-emosol**
domain looks spammy

### CODE REVIEW

**Comment by imme-emosol** [comment]
hmm, SimpleTest TAP reporter looks spammy too.

And Lens seems to not resolve.

File: producers.md [hunk 1]
```diff
@@ -485,10 +485,10 @@ testing frameworks and supports logging TAP.
...
 is a TAP producer for [SimpleTest](http://www.simpletest.org/).
 
-**[SnapTest](http://www.snaptest.net/)** is a powerful unit testing framework
-for PHP 5+, leveraging PHP's unique runtime language to simplify the unit
-test process without sacrificing the agility tests provide. It has a built-in
-TAP reporter.
+
+
+
+
```
Comment (kinow) [inline producers.md:491]: Can you delete the whitespace, please?

**Review by kinow — COMMENTED** [review]
Yeah, that domain does not look good. If you'd like to update your PR to include the other broken domains you found, that way we can do a big cleanup in one go :slightly_smiling_face: thanks!

File: producers.md (hunk 1 above)
Comment (imme-emosol) [reply producers.md]: Somehow backspace didn't work from the browser editor on my mobile, thanks for removing them.

**Pull request merged**